starter-code/pizza.js

```diff
@@ -1 +1,43 @@
 // Write your Pizza Builder JavaScript in this file.
+$(document).ready(function() {
+  var pep = 1;
+  var mush = 1;
+  var green = 1;
+  var sauce = 3;
+  var crust = 5;
+  var def = 13;
+  $(".price strong:nth-child(4)").html("$13");
+  //   strong.document.getElementsByTagName("STRONG").css.display = "none";
+
+  $(".price li:nth-child(4)").hide();
```

Comment:
It's a good idea to reset the pizza to a default state. Try maybe putting all of these lines in one function called `resetPizza` or something like that, and then just call it